readme.md

```diff
@@ -457,6 +457,7 @@ Check out my [blog](https://blog.sindresorhus.com) and follow me on [Twitter](ht
 - [ChatOps](https://github.com/exAspArk/awesome-chatops) - Managing technical and business operations through a chat.
 - [Falsehood](https://github.com/kdeldycke/awesome-falsehood) - Falsehoods programmers believe in.
 - [Domain-Driven Design](https://github.com/heynickc/awesome-ddd) - Software development approach for complex needs by connecting the implementation to an evolving model.
+- [Free Software](https://github.com/johnjago/Awesome-Free-Software) - Common applications that meet the free software criteria set by the FSF.
```

Comment:
Maybe something shorter like:

> Free as in freedom.

-

Would prefer if the repo name was lowercase, like all others.